api/server/router/system/backend.go

```diff
@@ -13,7 +13,7 @@ import (
 // Backend is the methods that need to be implemented to provide
 // system specific functionality.
 type Backend interface {
-	SystemInfo() (*types.Info, error)
+	SystemInfo() *types.Info
```

Comment:
Wondering if we should keep this a pointer or not

Reply:
Keep this a pointer +1. 
I tried to compare the difference between keep this a pointer or not with pprof, but the actual change is not big. So in theory, keep this a pointer may be more appropriate.